_posts/en/newsletters/2022-07-27-newsletter.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'Bitcoin Optech Newsletter #210'
+permalink: /en/newsletters/2022/07/27/
+name: 2022-07-27-newsletter
+slug: 2022-07-27-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Multiformat single-sig message signing:** Bitcoin Core and many
+  other wallets have long included support for signing and verifying
+  arbitrary messages when the key used to sign them corresponds to a
+  P2PKH address.  Bitcoin Core doesn't support signing or verifying
+  arbitrary messages for any other address type, including addresses
+  covering single-sig P2SH-P2WPKH, native P2WPKH, and P2TR outputs.  A
+  previous proposal, [BIP322][], to provide [fully generic message
+  signing][topic generic signmessage] that could work with any script
+  has not yet [been merged][bitcoin core #24058] into Bitcoin Core or
+  added to any other popular wallet of which we're aware.
+
+    This week, Ali Sherief [proposed][sherief gsm] that the same message
+    signing algorithm used for P2WPKH also be used for other output
+    types.  For verification, programs should infer how to derive the
+    key (if necessary) and verify the signature using the address type.
+    E.g., when provided a [bech32][topic bech32] address with a 20 byte
+    data element, assume it's for a P2WPKH output.
+
+    Developer Peter Gray of Coinkite [noted][gray cc] that ColdCard
+    wallets already create signatures in this way and developer Craig
+    Raw of Sparrow Wallet [said][raw sparrow] that wallet is able to
+    validate them in addition to also following the [BIP137][]
+    validation rules and a slightly different set of rules implemented
+    in Electrum.
+
+    Sherief is planning to write a BIP specifying the behavior.
+
+- **Proof of micro-burn:** several developers [discussed][pomb] how to
+  allow a single onchain transaction that destroys bitcoins ("burns"
+  bitcoins) be used in several different places for proving a subset of
+  those bitcoins had been burned.  To extend an example of Ruben
```

Comment:
This seems a bit circular to me, like "burn bitcoins to prove bitcoins have been burned"
Perhaps mention that both use cases and methods are discussed?
```suggestion
- **Proof of micro-burn:** several developers [discussed][pomb]
  use cases and designs of onchain transactions that destroy bitcoins
  ("burn" bitcoins) as a proof of resource consumption. To extend an example use case of Ruben
```